mkdocs.yml

```diff
@@ -78,6 +78,7 @@ nav:
     - Contributing: 'developers/contributing.md'
     - Development Tools: 'developers/development_tools.md'
     - Library Examples: 'developers/library_examples.md'
+    - Parallel Transfer: 'developers/parallel_transfer.md'
```

Comment:
Just playing around with where to put this information...

I would think it's more of a System Overview under a more general "Data Transport" (or Data Movement, or Data Transfer - whatever) page, or even under the Data Objects page. Thoughts?

Reply:
Hmm, I see what you mean. These docs feel like they could go in either location to me. As I was putting this together, it felt like it was more appropriate for a developer. That's mostly in regard to the "steps". The first two sections read more like something that should live under `System Overview > Data <pick_your_favorite>`.

Perhaps this should be split up. I like seeing everything in one spot though.

Thoughts?

Reply:
Agreed on keeping it all together. Should we choose to employ / implement some other data moving machinery in the future, it would fit here just as well. Perhaps where it is now is fine. Feel free to resolve if you agree.

Reply:
also could consider putting a pointer in the 'other' place pointing to this place.   all good.

Reply:
Just looked at the Data Objects page and it's starting to look like it could be a good spot for this.

Reply:
Moved to the data objects page.